src/lib/platform/exit_code.h

```diff
@@ -0,0 +1,152 @@
+/*
```

Comment:
Since we are making a platform independent library. I think we should avoid using code utilities just for posix platform. How about convert them into OpenThread's common code utilities?
What do you think? @jwhui @abtink

Reply:
Move to issue https://github.com/openthread/openthread/issues/4828